src/lib/validation.js

```diff
@@ -37,6 +37,14 @@ function validOptions(actual) {
     typeof actual.ignoreFunctions !== 'boolean' &&
     actual.ignoreFunctions !== null) return false
 
+  if ('disableFix' in actual &&
+    typeof actual.disableFix !== 'boolean' &&
+    actual.disableFix !== null) return false
+
+  if ('autofixFunc' in actual &&
```

Comment:
`autoFixFunc` should be either of:
- type `function` (called directly)
- or type `string` (turned into a function by `new Function()` and call subsequetly)

Reply:
Good news! We don't have to pass a string. function can be passed from stylelintrc.js config as it is
<img width="463" alt="passing autofixfunction" src="https://user-images.githubusercontent.com/6542274/39996276-99e122f2-579c-11e8-950d-fb54a3350092.png">

Reply:
Yeah, I know - stylelint is using [cosmiconfig](https://github.com/davidtheclark/cosmiconfig) 👍